examples/selenoid-example/codecept.conf.js

```diff
@@ -0,0 +1,28 @@
+exports.config = {
+  tests: './*_test.js',
+  output: './output',
+  helpers: {
+    WebDriver: {
+      url: 'http://localhost',
+      browser: 'chrome',
+    },
+  },
+
+  plugins: {
+    selenoid: {
+      enabled: true,
+      name: 'selenoid',
+      deletePassed: true,
+      autoCreate: true,
+      autoStart: true,
+      sessionTimeout: '30m',
```

Comment:
I don't think this timeout is using elsewhere in code?

Reply:
It is passed into Selenoid directly